packages/components-react/package.json

```diff
@@ -32,49 +32,49 @@
     "dist/"
   ],
   "dependencies": {
-    "clsx": "2.0.0",
-    "date-fns": "2.30.0",
+    "clsx": "2.1.0",
+    "date-fns": "3.3.1",
```

Comment:
It seems we don't actually use this. Should it be removed or are we keeping it for reasons?